build-lib/lib.js

```diff
@@ -108,7 +103,7 @@ window.ShopifyBuy = require('shopify-buy/shopify').default;
       ], { templateString: '{{versionString}}' })
     ]);
 
-    tree = mergeTrees([concatenatedScripts, nodeLibOutput]);
+    tree = mergeTrees([licensedAndVersionedTree, nodeLibOutput]);
```

Comment:
hey @minasmart, changed this during the rebase while resolving conflict. for review before merging